openslides/core/models.py

```diff
@@ -152,6 +163,25 @@ def need_full_update_for(self, collection_element):
         return True
 
 
+class ProjectionDefault(RESTModelMixin, models.Model):
```

Comment:
What is die advantage of splitting up this into two models Projector and ProjectionDefault? Even if we have two models on server side (which may be ok also for readability), you can combine them in the REST API. Only Projector may be the root rest model. (Remove access_permissions here and use get_root_rest_element(self): return self.projector.)